R/modify_pd.R

```diff
@@ -16,7 +16,7 @@ indent_round <- function(pd, indent_by) {
     start <- stop <- 0
   }
   pd <- pd %>%
-    mutate(indent = ifelse(seq_len(nrow(pd)) %in% start:stop, indent_by, 0)) %>%
+    mutate(indent = indent + ifelse(seq_len(nrow(pd)) %in% start:stop, indent_by, 0)) %>%
```

Comment:
At some point I thought it was important to work with relative indents (local to the current sub-tibble), and compose the final indent only during deserialzation. We can initializa `indent` with zero, but do we really need to work with absolute indent values?